src/core/service.js

```diff
@@ -52,10 +51,12 @@ function doCallEndpoint(callSpec, next, onReceive, onError, store) {
       // }
       (response.ok
         ? response.json().then(json => next(transformReceiveAction(onReceive((json)), store)))
+        // TODO: error parsing should be configurable globally (here we assume that a json object comes
+        // with { error: { message } }
         : response.json().then(json => next(onError(json.error.message))))
     )
     .catch(error => {
-      console.error('ERROR on API Call', error)
+      // console.error('ERROR on API Call', error)
```

Comment:
What is the proper error handling solution we want to implement here? Should this throw instead? Probably out of scope of this PR though

Reply:
good question, this all runs async' as a dispatch. We should understand how to handle it. Maybe throwing and then from your componente you can do `.catch()` Still I believe that 95% of the times you don't want to handle the error on components. If it is for UI we already provide the state on the placeholder prop to show an error in the UI. So probably no body will care about this (?)